lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
-      key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+    def display_flash_messages(**options)
```

Comment:
@jeffse forget I said anything: adding foundation 6 is a fundamentally breaking change so we should feel free to change other parts of the api where it makes sense.

The only thing I'd say is, since this is a potentially subtle breaking change (potentially a button might end up not being the expected colour), we should mention this somewhere - at least in the changelog. 

Or maybe we just need to flag it up to @potomak, who added the feature of having custom hashes in the first place and may be the only person using this feature 😜